File: Java/1019 - Conversão de Tempo.java
/* 
   ==UserScript==
 @name         Conversão de Tempo
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1019 - Conversão de Tempo
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/


import java.util.Scanner;
import java.io.IOException;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner scan = new Scanner(System.in);
        int segundos = scan.nextInt();
        int minutos = 0;
        int horas = 0;
        
        while(segundos>=60){
            minutos += 1;
            segundos -= 60;
        }
        
        while(minutos>=60){
            horas += 1;
            minutos -= 60;
        }
        
        System.out.println(String.format("%d:%d:%d", horas, minutos, segundos));
        
    }
 
}

File: C/1019 - Conversão de Tempo.c
/* 
   ==UserScript==
 @name         Conversão de Tempo
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1019 - Conversão de Tempo
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

#include <stdio.h>

int
main ()
{
  int segundos, minutos, horas;

  scanf ("%d", &segundos);

  horas = (segundos/60)/60;
  minutos = (segundos/60)%60;
  segundos = segundos - (segundos/60)*60;
  
  printf ("%d:%d:%d\n", horas, minutos, segundos);

  return 0;
}
